### DISCUSSION TITLE
Theme Preferences not syncing with System despite settings (Night Theme is active during the day)

### DISCUSSION THREAD

**Discussion by James-Firth** [community/community#53043] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am running Firefox 112.0.1 and MacOS Ventura 13.2.1, I have selected `Sync with system` as my Theme Mode under my GitHub Settings > Appearance > Theme Preferences.

However in the middle of the afternoon at 1pm local time, I am still seeing the `Night theme` as the active one. I would expect the `Day theme` to be the active one right now

![image](https://user-images.githubusercontent.com/485809/232581714-a9dbef23-e821-4157-81d7-2e4f0c04b945.png)

I believe I have seen GitHub's theme switch in the past, but it's not working today, despite restarting my browser to update.

**Comment by Dushistov** [comment]
Yeah, and on the same system where firefox + github can not sync theme,
safari + github works just fine.

**Comment by TyceHerrman** [comment]
Same issue. Also related discussion: #9305

**Comment by SubvertDev** [comment]
Same issue. Arc + macOS Sonoma.

**Reply by Sjmarf** [reply]
Same here. For me, it was because I had my Arc theme locked to dark mode. To fix it, right-click on the sidebar and choose "Edit Theme". Then click on the sparkles at the top of the popover.

![Screenshot 2024-06-05 at 12 18 10](https://github.com/community/community/assets/78750526/c22237f9-5b0e-46b2-8104-4452c461b6ab)

**Reply by erawhctim** [reply]
This was my issue; Arc was the root cause, not GitHub. Thanks!

**Reply by zano-spikerz** [reply]
@Sjmarf Was exactly my problem, thanks!

**Comment by James-Firth** [comment]
Tested a few things after seeing replies on this.

Interesting results, I'm not sure if this is a Firefox issue or a Github issue now.

I confirmed Firefox is set to sync with my system (which uses sunrise/sunset for themes)
![image](https://github.com/community/community/assets/485809/3b6bd18e-3ebd-4e3f-9c66-caede65fe695)

However, I tried switching my Firefox theme from `Dark` to `System theme - auto` and once I did _that_, now Firefox and Github are in sync.
![image](https://github.com/community/community/assets/485809/2cd8227f-3de2-4b48-9c35-2137a6a18b08)


This seems like odd behaviour and makes me lean towards it being a Firefox issue? But I'm not 100% sure.
[I found this Mozilla knowledge base link](https://support.mozilla.org/en-US/kb/change-website-appearance-settings-firefox) and it makes me think that Github should be using the `Website appearance`, but it's a bit unclear to me right now.

**Reply by Michael21107** [reply]
So both your system and Firefox are running the correct themes, i.e. not dark theme at 1 pm?
